setup.cfg

```diff
@@ -23,7 +23,6 @@ python_requires = >=3.7
 install_requires =
     numpy
     matplotlib
-    shapely
```

Comment:
Did you remove the shapely altogether? 😮

Reply:
Ha, whoops!  I blindly hit "merge" on your suggestions without paying attention.  I will fix it, one moment...

Reply:
Yikes, so I am to blame! 😬